config/helmchart/values.yaml.tpl

```diff
@@ -16,19 +16,19 @@ fullnameOverride: ""
 
 serviceAccount:
   # Specifies whether a service account should be created
-  create: true
+  create: false
   # Annotations to add to the service account
   annotations: {}
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: ""
+  name: "default"
```

Comment:
why this change? and the one above?

Reply:
It will only work if using the default serviceaccount right now (rolebindings from kustomization use default).